pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "thoth-common"
+description = "Shared code logic in the project Thoth."
+readme = "README.rst"
+authors = [
+    { name = "Fridolin Pokorny", email = "fridolin@redhat.com" }
+]
+license = { text = "GPLv3+" }
+dynamic = [ "version", "dependencies" ]
+
+[tool.setuptools]
+zip-safe = false
+packages = ["thoth.common", "thoth.common.config"]
+
+[tool.setuptools.dynamic]
+version = {attr = "thoth.common.__version__"}
+dependencies = { file = ["requirements.txt"] }
+
+[tool.setuptools.package-data]
+thoth.common = ["py.typed"]
```

Comment:
Since key name contain special character `(.)` it should be enclosed with double quotes `(")` eg: `"thoth.common" = ["py.typed"]`

Reply:
Resolved